barretenberg/cpp/src/barretenberg/ultra_honk/ultra_verifier.cpp

```diff
@@ -49,7 +49,7 @@ template <typename Flavor> bool UltraVerifier_<Flavor>::verify_proof(const HonkP
     transcript = std::make_shared<Transcript>(proof);
     VerifierCommitments commitments{ key };
     OinkVerifier<Flavor> oink_verifier{ key, transcript };
-    auto [relation_parameters, witness_commitments] = oink_verifier.verify();
+    auto [relation_parameters, witness_commitments, public_inputs] = oink_verifier.verify();
```

Comment:
seems like public_inputs isn't used by ultra_verifier so maybe this can just be _

Reply:
the public inputs are used by the verifier to compute the public_input_delta which i believe is now computed by OinkVerifier so maybe the Oink output shouldn't even include them

Reply:
they're included because the protogalaxy verifier needs to fold them

Reply:
Right yeah